dd-java-agent/agent-tooling/src/main/java/datadog/trace/agent/tooling/TracerInstaller.java

```diff
@@ -14,6 +14,10 @@ public static synchronized void installGlobalTracer() {
     if (!io.opentracing.util.GlobalTracer.isRegistered()) {
       Configuration conf = Configuration.fromEnv();
 
+      if (Strings.isNullOrEmpty(conf.getReporter().getSenderConfiguration().getAuthUsername())) {
+        conf.getReporter().getSenderConfiguration().withAuthUsername("auth");
```

Comment:
Shouldn't this only happen if a token/password is also set? Or is the sender correctly ignoring authentication when no token/password is configured?

Reply:
It does but I will conditionalize this to make it cleaner.